util/build_configs/cray-internal/setenv-xc-aarch64.bash

```diff
@@ -284,6 +284,10 @@ else
         # load target PrgEnv with compiler version
         load_module $target_prgenv
         load_module_version $target_compiler $target_version
+
+        # pin an older version of MPICH that works with gen_version_gcc
+        unload_module cray-mpich
+        load_module cray-mpich/7.7.13
```

Comment:
I would do `load_module_version cray-mpich 7.7.13`

That will take care of the unload and log the swap.